server/src/controllers/orderController.ts

```diff
@@ -32,9 +32,21 @@ orderController.post(
               object.payment_intent,
               object.id,
           );
```

Comment:
It looks like the donation is not being handled when it is made during checkout, should that be added to the webhook or should that option be entirely removed from the checkout process?

Reply:
Donation process should be separate from ticket purchasing. I understand that it would be nice to buy a ticket AND donate at the same time, but we could never get that working correctly, unfortunately. If we can split them, I'm more than happy to keep looking into making it possible to buy tickets and donate in the same transaction.

Reply:
Currently the checkout process has an option to tack on a donation to a ticket purchase so that might need to be removed.

Reply:
We will want want to have the ability to both make a donation while purchasing a ticket (in the same transaction) and make a separate donation. If we have to split the webhook or make a new one to support both cases then that should be fine

Reply:
I can take a look at incorporating the donation record creation with the ticketing webhook session type.

Reply:
That would be great! If you do, would you mind creating a ticket on the project board?